typescript/cli/src/commands/warp.ts

```diff
@@ -93,15 +102,34 @@ export const apply: CommandModuleWithWriteContext<{
       process.exit(0);
     }
     const warpDeployConfig = await readWarpRouteDeployConfig(config);
+    let chainSubmissionStrategy;
+    if (strategyUrl) {
+      chainSubmissionStrategy = readChainSubmissionStrategy(strategyUrl);
+    }
     await runWarpRouteApply({
       context,
       warpDeployConfig,
       warpCoreConfig,
+      chainSubmissionStrategy,
     });
     process.exit(0);
   },
 };
 
+/**
+ * Retrieves a chain submission strategy from the provided filepath.
+ * @param submissionStrategyFilepath a filepath to the submission strategy file
+ * @returns a formatted submission strategy
+ */
+export function readChainSubmissionStrategy(
```

Comment:
I still think we should keep these `command/*` files void of business logic, just the HCI stuff. So suggest moving this

Reply:
I agree. will fix

Reply:
Moved this particular fn out. The other parameter related branching above this is a bigger lift so i'll leave them for now